### PR TITLE
release-0.24: ci: fix wrong image TAG for kind tests

### PR DESCRIPTION
**Pull request by mythi**
Signed-off-by: Mikko Ylinen <mikko.ylinen@intel.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -151,7 +151,7 @@ pipeline {
         stage('make test-with-kind') {
           steps {
             dir(path: "$REPO_DIR") {
-              sh "make test-with-kind REG=intel/ TAG=devel"
+              sh "make test-with-kind REG=intel/ TAG=0.24.0"
             }
```
Comment (bart0sh) [inline Jenkinsfile:155]: Does it mean that CI will test only the latest release?

**Review by bart0sh — APPROVED** [review]
/lgtm

**Comment by mythi** [comment]
@bart0sh this is for the release branch. `make test-with-kind` builds images with `devel` tag but the test case deployment looks for `0.24.0`.

**Comment by mythi** [comment]
@bart0sh let's merge

**Pull request merged**